MauiMacAppWithBroker.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why do you need a new sln file? We already load a ton of dev apps by default in LibsAndSamples.sln and you can set a solution config to ignore the maui app (happy to show you how to). Is it because our CI build doesn't build your maui app?

Not saying that a new sln file is a bad approach, just trying to figure out what you want to do.

Reply:
Per offline discussion, if the new MAUI app added to LibsAndSamples.sln, building on mac will fail due to compatibly issues. Added a new Mac.sln for mac broker sample apps.